charts/mageai/templates/serviceaccount.yaml

```diff
@@ -36,3 +36,26 @@ roleRef:
   kind: Role # This must be Role or ClusterRole
   name: job-manager # This must match the name of the Role or ClusterRole you wish to bind to
   apiGroup: rbac.authorization.k8s.io
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: {{ include "mageai.fullname" . }}-workspace-manager
+rules:
+- apiGroups: ["", "apps"] # "" indicates the core API group
+  resources: ["nodes", "persistentvolumeclaims", "services", "statefulsets"]
+  verbs: ["get", "list", "watch", "create", "update", "patch", "delete"]
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: {{ include "mageai.fullname" . }}-workspace-manager
```

Comment:
what's `mageai.fullname` here?

Reply:
That just gets a prefix for the role name. The default prefix is mageai. Have a look at the _helper.yaml file for the definition.